test/dotnet.Tests/CommandTests/Run/RunFileTests.cs

```diff
@@ -807,6 +807,47 @@ public void NoBuild_02()
             .And.HaveStdOut("Changed");
     }
 
+    [Fact]
```

Comment:
nit: We have a `LinuxOnlyFact` if you want to use that instead. Or a `PlatformSpecificFact` that takes an or'ed set of platform flags.

Reply:
Didn't find `LinuxOnlyFact` actually, but `PlatformSpecificFact(TestPlatforms.AnyUnix)` looks good, thanks.